starter-code/index.js

```diff
@@ -1,10 +1,38 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+    this.items = [];
+    this.length = 0;
+};
+
+SortedList.prototype.add = function(item) {
+    this.items.push(item);
+    this.length++;
+    this.items.sort(function(item1,item2){
+        return item1 - item2;
+    })
+}
+
+SortedList.prototype.get = function(pos) {
+    return this.items[pos - 1];
+}
+
+SortedList.prototype.max     = function() {
+    return this.items[this.length - 1];
+}
+
+SortedList.prototype.min     = function() {
+    return this.items[0];
+}
+
+SortedList.prototype.average = function() {
+    return this.items.reduce(function(acc, item , index , array){
+        return index < array.length - 1  ? acc + item : (acc + item)/ array.length
+    },0);
+}
+
```

Comment:
Por qué complicáis tanto la operación del `reduce`? Es menos complicado si cuando acabe el reduce justo dividís entre la longitud (`this.length`). Es decir `return this.items.reduce(() => lo_que_sea) / this.length || this.items.length`